[15.0][IMP] subscription_oca: Add previous contributors

carlos-domatix opened the pull request:
Sorry, previous contributors forgotten in https://github.com/OCA/contract/pull/971I add them.

@simahawk

BurkhalterY committed:
[MIG] contract_queue_job: Migration to 14.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 62.8% (22 of 35 strings)

Translation: contract-14.0/contract-14.0-contract_variable_quantity
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_variable_quantity/it/

OCA-git-bot committed:
Merge PR #897 into 14.0

Signed-off-by etobella

Commit:
[UPD] Update contract_queue_job.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (49 of 49 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (49 of 49 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.1% (313 of 319 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

OCA-git-bot committed:
Merge PR #940 into 14.0

Signed-off-by rousseldenis

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
contract 14.0.2.9.5

OCA-git-bot committed:
[UPD] addons table in README.md

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 63.1% (48 of 76 strings)

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 98.1% (313 of 319 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 63.1% (48 of 76 strings)

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 98.1% (313 of 319 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

mourad-ehm committed:
[IMP][14.0] refactor code of product_contract module (make easy inherit in other modules)

mourad-ehm committed:
[FIX][14.0] tests & typo

stenext committed:
[IMP] Add INVOICEMONTHNAME marker to line description

mymage committed:
Translated using Weblate (Italian)

Currently translated at 63.1% (48 of 76 strings)

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/it/

OCA-git-bot committed:
Merge PR #946 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update contract.pot

OCA-git-bot committed:
contract 14.0.2.10.0

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

niyasraphy committed:
[FIX] contract: show uom only when multi uom is activated

before this commit, the unit of measure field is
always visible in different views, irrespective
of the unit of measure option in the settings.

after this commit, the unit of measure field will
be shown only if the multi uom feature is activated
in the settings.

OCA-git-bot committed:
Merge PR #953 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
contract 14.0.2.11.0

OCA-git-bot committed:
[UPD] addons table in README.md

Commit:
Fix #INVOICEMONTHNAME# for all languages

It was working only in English

- Changed %B to %m to work on all languages
- Update month translations in es.po
- Add the possibility to replace #INVOICEMONTHNUMER# and #INVOICEYEAR# in line descriptions

OCA-git-bot committed:
Merge PR #954 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
contract 14.0.2.11.1

OCA-git-bot committed:
[UPD] addons table in README.md

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (333 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/es_AR/

damdam-s committed:
[14.0][ADD] contract_split module

mymage committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

damdam-s committed:
set default value of split quantity to initial contract line quantity

damdam-s committed:
IMP: add roadmap

damdam-s committed:
refactor module

victoralmau committed:
[FIX] contract: Set fixed date to check test correctly.

We need to set a fixed date for teststest_check_month_name_marker to check against
the month of the date we are set (otherwise it would check against the month of today).

Commit:
Translated using Weblate (Catalan)

Currently translated at 100.0% (76 of 76 strings)

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/ca/

OCA-git-bot committed:
Merge PR #966 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
contract 14.0.2.11.2

OCA-git-bot committed:
[UPD] addons table in README.md

damdam-s committed:
fix test import and use uom rouding for float_compare

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (49 of 49 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (333 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/pt/

Commit:
[IMP ] #INVOICEMONTHNUMBER# #INVOICEYEAR#

ADD in the view form , in Legend , to use it on descriptions

#INVOICEMONTHNUMBER#: Invoice month number of the period
#INVOICEYEAR#: Invoice year of the period

OCA-git-bot committed:
Merge PR #821 into 14.0

Signed-off-by leemannd

OCA-git-bot committed:
contract_payment_mode 14.0.1.1.2

OCA-git-bot committed:
contract 14.0.2.11.3

OCA-git-bot committed:
contract_mandate 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #945 into 14.0

Signed-off-by dreispt

Commit:
[UPD] Update product_contract.pot

OCA-git-bot committed:
product_contract 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/

OCA-git-bot committed:
Merge PR #967 into 14.0

Signed-off-by dreispt

Commit:
[UPD] Update contract_split.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[ADD] icon.png

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 63.1% (48 of 76 strings)

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 92.1% (47 of 51 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 92.1% (47 of 51 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 63.1% (48 of 76 strings)

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/it/

OCA-git-bot committed:
[UPD] README.rst

mymage committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

marcelsavegnago committed:
[FIX] product_contract: fixes the visibility of the is_contract field in the sales order form

carlos-domatix committed:
[15.0][ADD] subscription_oca

OCA-git-bot committed:
Merge PR #987 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update subscription_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

ilyasProgrammer committed:
[IMP] subscription_oca: tests

ilyasProgrammer committed:
[IMP] subscription_oca: refactoring

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] contract: Apply some values directly in Form

If not applied, they will not be taken into account in the lines
operations.

We have needed to add an auxiliary form that adds the fields without
groups.

TT45329

OCA-git-bot committed:
Merge PR #995 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

ilyasProgrammer committed:
[MIG] subscription_oca: Migration to 14.0

ilyasProgrammer committed:
[IMP] subscription_oca: tests

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract_sale
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_sale/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_sale
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_sale/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_serviceprofile
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_serviceprofile/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract_payment_mode
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_payment_mode/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_stock
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_stock/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract_mandate
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_mandate/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_project
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_project/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_mrp
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_mrp/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_maintenance
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_maintenance/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_legal_sale
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_legal_sale/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_repair
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_repair/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract_variable_quantity
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_variable_quantity/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_legal
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_legal/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract_sale_generation
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_sale_generation/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract_invoice_start_end_dates
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_invoice_start_end_dates/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (330 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

ilyasProgrammer committed:
[IMP] subscription_oca: add maintainers

OCA-git-bot committed:
Merge PR #999 into 14.0

Signed-off-by simahawk

Commit:
[UPD] Update subscription_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (171 of 171 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 79.8% (266 of 333 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/pt_BR/

marcelsavegnago committed:
[FIX] contract: fix domain of payment_term_id field

damdam-s committed:
IMP: use group product.group_discount_per_so_line instead of base.group_no_one

damdam-s committed:
add optional attribute on discount field of contract.line

OCA-git-bot committed:
[IMP] update dotfiles

Commit:
[UPD] Update subscription_oca.pot

Commit:
[UPD] Update agreement.pot

Commit:
[UPD] Update agreement_serviceprofile.pot

Commit:
[UPD] Update contract.pot

Commit:
[UPD] Update agreement_tier_validation.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_tier_validation
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_tier_validation/

pedrobaeza committed:
[FIX] *: Linter fixes

Commit:
[UPD] Update agreement_legal.pot

OCA-git-bot committed:
Merge PR #1014 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_serviceprofile
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_serviceprofile/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_legal
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_legal/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/

OCA-git-bot committed:
Merge PR #1012 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 99.0% (329 of 332 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

OCA-git-bot committed:
Merge PR #974 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update contract.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (170 of 170 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

barkat-matthias committed:
Added translation using Weblate (French (France) (fr_FR))

barkat-matthias committed:
Translated using Weblate (French (France) (fr_FR))

Currently translated at 100.0% (29 of 29 strings)

Translation: contract-14.0/contract-14.0-contract_split
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_split/fr_FR/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (170 of 170 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (265 of 265 strings)

Translation: contract-14.0/contract-14.0-agreement_legal
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_legal/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (170 of 170 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (20 of 20 strings)

Translation: contract-14.0/contract-14.0-agreement_stock
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_stock/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (15 of 15 strings)

Translation: contract-14.0/contract-14.0-contract_sale_generation
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_sale_generation/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (20 of 20 strings)

Translation: contract-14.0/contract-14.0-agreement_tier_validation
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_tier_validation/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: contract-14.0/contract-14.0-contract_update_last_date_invoiced
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_update_last_date_invoiced/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: contract-14.0/contract-14.0-agreement_sale
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_sale/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (15 of 15 strings)

Translation: contract-14.0/contract-14.0-agreement_maintenance
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_maintenance/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 79.3% (23 of 29 strings)

Translation: contract-14.0/contract-14.0-contract_split
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_split/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (29 of 29 strings)

Translation: contract-14.0/contract-14.0-contract_split
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_split/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: contract-14.0/contract-14.0-agreement_account
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_account/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: contract-14.0/contract-14.0-agreement_repair
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_repair/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: contract-14.0/contract-14.0-agreement_project
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_project/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (75 of 75 strings)

Translation: contract-14.0/contract-14.0-agreement
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: contract-14.0/contract-14.0-agreement_legal_sale
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_legal_sale/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: contract-14.0/contract-14.0-agreement_mrp
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_mrp/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (334 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (51 of 51 strings)

Translation: contract-14.0/contract-14.0-agreement_serviceprofile
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_serviceprofile/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (51 of 51 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: contract-14.0/contract-14.0-contract_delivery_zone
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_delivery_zone/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: contract-14.0/contract-14.0-contract_sale_tag
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_sale_tag/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: contract-14.0/contract-14.0-contract_mandate
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_mandate/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: contract-14.0/contract-14.0-contract_variable_qty_prorated
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_variable_qty_prorated/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: contract-14.0/contract-14.0-contract_payment_mode
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_payment_mode/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: contract-14.0/contract-14.0-contract_queue_job
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_queue_job/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: contract-14.0/contract-14.0-contract_invoice_start_end_dates
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract_invoice_start_end_dates/es/

scigghia committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

renda-dev committed:
[FIX] subscription_oca: module icon

PicchiSeba committed:
[IMP] subscription_oca: enable markers to print invoice names

(like in _contract_)

PicchiSeba committed:
[FIX] subscription_oca: adjust broken test checks

PicchiSeba committed:
[FIX] subscription_oca: adjust broken test checks

PicchiSeba committed:
[IMP]subscription_oca: enable additional filters for `sale.subscription`

OCA-git-bot committed:
Merge PR #1034 into 14.0

Signed-off-by ilyasProgrammer

OCA-git-bot committed:
[BOT] post-merge updates

PicchiSeba committed:
[IMP] subscription_oca: add maintainers

OCA-git-bot committed:
Merge PR #1036 into 14.0

Signed-off-by ilyasProgrammer

OCA-git-bot committed:
[BOT] post-merge updates

ilyasProgrammer committed:
[IMP] subscription_oca: more invoicing modes

OCA-git-bot committed:
Merge PR #1028 into 14.0

Signed-off-by aleuffre

OCA-git-bot committed:
Merge PR #1035 into 14.0

Signed-off-by aleuffre

Commit:
[UPD] Update subscription_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 99.4% (174 of 175 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (175 of 175 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

renda-dev committed:
[IMP] subscription_oca: modify tree view

- Add start date
- Add finish date
- Change 'Etapa' string to 'Stage'

OCA-git-bot committed:
Merge PR #1039 into 14.0

Signed-off-by aleuffre

Commit:
[UPD] Update subscription_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

OCA-git-bot committed:
Merge PR #1031 into 14.0

Signed-off-by renda-dev

Commit:
[UPD] Update subscription_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (192 of 192 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

OCA-git-bot committed:
Merge PR #1037 into 14.0

Signed-off-by aleuffre

Commit:
[UPD] Update subscription_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (199 of 199 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

GSLabIt committed:
[FIX] subscription_oca: better partner form inheriting

OCA-git-bot committed:
Merge PR #1043 into 14.0

Signed-off-by ilyasProgrammer

Commit:
[UPD] Update agreement_tier_validation.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-agreement_tier_validation
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_tier_validation/

PicchiSeba committed:
[IMP]subscription_oca: search products by name, category and attributes

OCA-git-bot committed:
Merge PR #1046 into 14.0

Signed-off-by PicchiSeba

OCA-git-bot committed:
[BOT] post-merge updates

PicchiSeba committed:
[IMP]subscription_oca: readonly company on sale_subscription, template must have same company or none

OCA-git-bot committed:
Merge PR #1011 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 92.1% (47 of 51 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

OCA-git-bot committed:
Merge PR #1047 into 14.0

Signed-off-by renda-dev

Commit:
[UPD] Update subscription_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (200 of 200 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (21 of 21 strings)

Translation: contract-14.0/contract-14.0-agreement_tier_validation
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-agreement_tier_validation/es/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (200 of 200 strings)

Translation: contract-14.0/contract-14.0-subscription_oca
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-subscription_oca/it/

HekkiMelody committed:
[FIX] subscription_oca: product-subscription_template company issues

Make company required on subscription_template.

Make subcription_template_id a company-dependent field
on product_template.

HekkiMelody committed:
[FIX] sale_subscription: get sales_team with consistent company on SO

OCA-git-bot committed:
Merge PR #1060 into 14.0

Signed-off-by aleuffre

OCA-git-bot committed:
[BOT] post-merge updates

rvalyi commented:
@carlos-domatix may be you should ping the recent contributors to approve this...

mymage committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (329 of 334 strings)

Translation: contract-14.0/contract-14.0-contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-contract/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 92.1% (47 of 51 strings)

Translation: contract-14.0/contract-14.0-product_contract
Translate-URL: https://translation.odoo-community.org/projects/contract-14-0/contract-14-0-product_contract/it/
